src/EventStore.Client.PersistentSubscriptions/PersistentSubscriptionInfo.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+using EventStore.Client.PersistentSubscriptions;
+using Google.Protobuf.Collections;
+
+namespace EventStore.Client {
+	/// <summary>
+	/// Provides the details for a persistent subscription
+	/// </summary>
+	/// <param name="EventSource">Name of the stream used for the persistent subscription</param>
+	/// <param name="GroupName">Name of the persistent subscription</param>
+	/// <param name="Status">Current state</param>
+	/// <param name="AveragePerSecond">The average amount of messages being processed per second</param>
```

Comment:
**FIX:** RFC states that stats should be grouped into the stats object. See https://github.com/EventStore/architecture-and-planning/blob/main/rfcs/022%20-%20gRPC%20Persistent%20Subscriptions.md#persistentsubscriptioninfo.